mainwindow.h

```diff
@@ -23,19 +23,20 @@
 
 enum WorkingModes
 {
-    MANUAL_MODE = 0,
+// QA: why you declare MANUAL_MODE, it will be the same 
+// QA: if you won't declare it, just MANUL_MODE, ...
+    MANUAL_MODE = 0, 
     SEMIAUTO_MODE,
     FULLAUTO_MODE
 };
 
 namespace Ui {
 class MainWindow;
 }
-
+// QA: Why not smart pointers insted of raw pointers? 
```

Comment:
Because I'm totally dumb in this topic

Reply:
You can learn ... 
This link https://www.internalpointers.com/post/beginner-s-look-smart-pointers-modern-c is pretty good (=